src/Employer/Employer.Web/Controllers/ApplicationReviewController.cs

```diff
@@ -55,6 +56,21 @@ public async Task<IActionResult> ApplicationReview(ApplicationReviewEditModel ed
                 return RedirectToRoute( RouteNames.VacancyManage_Get, new { editModel.EmployerAccountId, editModel.VacancyId, vacancySharedByProvider });
             }
 
+            if (editModel.Outcome == ApplicationReviewStatus.InReview || editModel.Outcome == ApplicationReviewStatus.Interviewing)
+            {
+                var applicationReviewStatusEditModel = new ApplicationReviewStatusConfirmationEditModel 
+                {
+                    ApplicationReviewId = editModel.ApplicationReviewId,
+                    VacancyId = editModel.VacancyId,
+                    EmployerAccountId = editModel.EmployerAccountId,
+                    CandidateFeedback = editModel.CandidateFeedback,
+                    Outcome = editModel.Outcome,
+                };
+                var candidateName = await _orchestrator.PostApplicationReviewConfirmationEditModelAsync(applicationReviewStatusEditModel, User.ToVacancyUser());
+                TempData.Add(TempDataKeys.ApplicationReviewStatusChangeInfoMessage, string.Format(InfoMessages.ApplicationStatusChangeBannerHeader, candidateName, editModel.Outcome.GetDisplayName().ToLower()));
+                return RedirectToRoute(RouteNames.VacancyManage_Get, new { editModel.VacancyId, editModel.EmployerAccountId });
+            }
```

Comment:
I feel like this is missing some tests 😁

Reply:
added one now